ekf2: migrate uorb events to events interface

bresch opened the pull request:
<!--

Thank you for your contribution!

Get early feedback through
- Dronecode Discord: https://discord.gg/dronecode
- PX4 Discuss: http://discuss.px4.io/
- opening a draft pr and sharing the link

-->

After this, the objective is to remove the intermediate flags and directly send the event where it occurs in the EKF2 backend.

bresch committed:
ekf2: migrate uorb events to events interface

junwoo091400 commented:
Just curious, how did it get added to the EKF project board? Is it automatic if you apply the EKF2 label?

bresch commented:
> Just curious, how did it get added to the EKF project board? Is it automatic if you apply the EKF2 label?

No, I added it to the board manually when creating the PR

dagar commented:
Should we bring this in for v1.14 or after?

mrpollo commented:
@bresch @dagar are we bringing this in now?

bresch commented:
Yes, we should I guess

bresch committed:
EKF2: send event when yaw gets aligned

dakejahl committed:
Merge branch 'main' into pr-ekf2-events

dakejahl reviewed (dismissed):
Do we want to bring this in? Any impact to GCS UX?

dakejahl committed:
ekf2: update ITCM linker scripts for renamed SendEvents

Update mangled symbol from PublishEventFlags to SendEvents in
ITCM function include linker scripts for fmu-v6xrt, mr-tropic,
and tropic-community boards.

dakejahl committed:
Merge branch 'main' into pr-ekf2-events
